### PR TITLE
Add CATALOG_SOURCE env Varaiable to install operator from other-catalog sources

### PR DESCRIPTION
**Pull request by praveen4g0**
* Now it adds flexibility to install p12n (pre-stage) operator where catalog-source is other than redhat-operators

### CODE REVIEW

File: env/test/test.properties [hunk 1]
```diff
@@ -2,4 +2,5 @@ CHANNEL = ocp-4.6
 OPERATOR_ENV = prod
 CSV_VERSION = v1.0.1
 INSTALL_OPERATOR = no
-SECRET_TOKEN = 1234567
\ No newline at end of file
+SECRET_TOKEN = 1234567
+CATALOG_SOURCE = pre-stage-operators
```
Comment (ppitonak) [inline env/test/test.properties:6]: When/where is this value used?

Comment (ppitonak) [reply env/test/test.properties:6]: `pkg/config/config.go` mentions `redhat-operators` but that is definitely not a default value and actually, I would prefer it to be default

Comment (praveen4g0) [reply env/test/test.properties:6]: sorry it's typo in `pkg/config/config.go` usually it gets value by default as `pre-stage-operators` but user can explicitly specify it's value at runtime if he want to use `redhat-operators`

my question here is what to default here??
`CATALOG_SOURCE = pre-stage-operators or redhat-operators`

mostly as we are using release-tests to test against pre-stage build it's safe to assume default as `pre-stage-operators`

File: pkg/config/config.go [hunk 2]
```diff
@@ -100,6 +101,10 @@ func initializeFlags() *EnvironmentFlags {
 	flag.StringVar(&f.Channel, "channel", defaultChannel,
 		"Provide channel to subcribe your operator you'd like to use for these tests. By default `canary` will be used.")
 
+	defaultCatalogSource := os.Getenv("CATALOG_SOURCE")
+	flag.StringVar(&f.CatalogSource, "catalogsource", defaultCatalogSource,
+		"Provide defaultCatalogSource to subscribe operator from. By default `pre-stage-operators` will be used.")
```
Comment (praveen4g0) [inline pkg/config/config.go:106]: fixed typo

File: env/test/test.properties (hunk 1 above)
Comment (ppitonak) [reply env/test/test.properties:6]: OK, let's leave it as is for now, we can re-evaluate later.

**Pull request merged**